reference/auth/models/consent/consent.v1.yaml

```diff
@@ -0,0 +1,30 @@
+title: Consent
+description: Consent
+type: object
+properties:
+  type:
+    description: Test
+    type: string
```

Comment:
Consider externalizing this to a separate model that can be re-used in the URL paths and here as well.

Since this is basically an enum, consider also adding a regex pattern of allowable characters and/or min/max length.